src/dev/impl/DevToys/Helpers/CertificateHelper.cs

```diff
@@ -68,6 +70,38 @@ internal static bool TryDecodeCertificate(string input, string? password, out st
                 return false;
             }
 
+            // Try to decode the X.509 extensions. Put this is a separate try/catch because it's not critical.
+            try
+            {
+                decoded += Environment.NewLine;
+
+                foreach (X509Extension x509Extension in certificate.Extensions)
+                {
+                    StringBuilder extensionData = new StringBuilder();
```

Comment:
I believe this `StringBuilder` could be declared outside of the `foreach` loop and it would be more optimal since right now you are allocating a new string on each iteration by concatenating to `decoded` ([line 95](https://github.com/veler/DevToys/pull/831/files#diff-b0f9ffa47ec01cd1ab78444903b160769b3c781f18cd59f380ea3b7b8331f7edR95)).

Reply:
Also, could you please fix the compiler warnings:
![image](https://github.com/veler/DevToys/assets/74258557/28dab6c9-bc80-40b0-93c9-7e3dba412911)

Reply:
> I believe this `StringBuilder` could be declared outside of the `foreach` loop and it would be more optimal since right now you are allocating a new string on each iteration by concatenating to `decoded` ([line 95](https://github.com/veler/DevToys/pull/831/files#diff-b0f9ffa47ec01cd1ab78444903b160769b3c781f18cd59f380ea3b7b8331f7edR95)).

You are correct!

> could you please fix the compiler warnings

Sure! I could change `StringBuilder` to `var`, or (my preference) I could use `new()` on the rhs. I couldn't find any guidance in [CONTRIBUTING](https://github.com/veler/DevToys/blob/main/CONTRIBUTING.md), so hopefully the latter is ok.

Reply:
457bd325d0b47555d917b6530109b254ca108bea

Reply:
I personally try and look at what is already in the project.  In this case, when searching for `= new();`, I see a ton of examples already there so this should be just fine.  I also like this syntax and make use of it in other projects.

Reply:
Oh, I will take comment as a sign that I should probably improve the CONTRIBUTING 😅